src/objects/zcl_abapgit_object_fugr.clas.abap

```diff
@@ -1,8 +1,11 @@
 CLASS zcl_abapgit_object_fugr DEFINITION PUBLIC INHERITING FROM zcl_abapgit_objects_program FINAL.
 
   PUBLIC SECTION.
-    INTERFACES zif_abapgit_object.
-    ALIASES mo_files FOR zif_abapgit_object~mo_files.
+    INTERFACES:
+      zif_abapgit_object,
+      zif_abapgit_lxe_texts.
```

Comment:
you can remove `zif_abapgit_lxe_texts` since no interface methods are implemented in this class

Reply:
Done